src/components/VContentSwitcher/meta/VContentSwitcherPopover.stories.mdx

```diff
@@ -0,0 +1,37 @@
+import {
+  ArgsTable,
+  Canvas,
+  Description,
+  Meta,
+  Story,
+} from '@storybook/addon-docs'
+
+import VContentSwitcherPopover from '@/components/VContentSwitcher/VContentSwitcherPopover.vue'
+
+<Meta
+  title="Components/VContentSwitcherPopover"
+  components={VContentSwitcherPopover}
+  argTypes={{
+    select: {
+      action: 'select',
+    },
+  }}
+/>
+
+export const Template = (args, { argTypes }) => ({
+  template: `<VContentSwitcherPopover active-item="image" v-bind="$props" v-on="$props"/>`,
+  components: { VContentSwitcherPopover },
+  props: Object.keys(argTypes),
```

Comment:
```suggestion
  template: `<VContentSwitcherPopover :active-item="activeItem" @select="setActiveItem" v-bind="$props" v-on="$props"/>`,
  components: { VContentSwitcherPopover },
  props: Object.keys(argTypes),
  data() {
    return {
      activeItem: 'image',
    }
  },
  methods: {
    setActiveItem(item) {
      this.activeItem = item
    }
  }
```

Reply:
Skipping in favour of the Actions tab showing emitted events.